src/form-builder/DateField.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useField, FieldHookConfig } from 'formik';
+
+import { FieldProps } from './types';
+import { getMessage } from '../utils/i18n';
+import {
+  ValidationFunction,
+  chainValidations,
+  required,
+} from '../utils/validation';
+
+import { SimpleDate } from '@department-of-veterans-affairs/component-library/Date';
+
+export type ComponentLibraryDatePart = {
+  value: string;
+  dirty: boolean;
```

Comment:
This does not spark joy.

Reply:
Agreed. I can probably remove it from here, honestly. With using `SimpleDate`, I don't actually _need_ it. It's never used.

Reply:
Would removing it cause more confusion?

Reply:
Maybe not. I'll try removing it and see if stuff breaks. :sweat_smile: